cloudsecrets/aws.py

```diff
@@ -39,28 +37,27 @@ class Secrets(SecretsBase):
     The Secrets class supports being called as a dictionary
     >>> dict(s).get("MYSECRET")
     'VALUE'
-    
-    """
-    def __init__(self,secret,**kwargs) -> None:
-        super().__init__(secret,**kwargs)
 
-        self._region = kwargs.get('region',None)
-
-        self.session = boto3.session.Session()
-        self.client = self.session.client(service_name='secretsmanager')
+    """
 
+    def __init__(self, secret, **kwargs) -> None:
+        super().__init__(secret, **kwargs)
+        connection = kwargs.get("connection", None)
```

Comment:
This was added in case a caller already had a handle to the expensive connection object.